.gitignore

```diff
@@ -0,0 +1 @@
+bin-install\
```

Comment:
Not against creating a gitignore (probably about time, have one uncommitted locally).

But this folder is currently part of any "official" workflow. If you delete the readme, maybe ignoring `*.xpi` files would be enough?

Reply:
deleting files and folder